packages/synapse-react-client/test/components/download_list/FolderDownloadConfirmation.test.tsx

```diff
@@ -11,7 +11,7 @@ import {
 import * as DownloadConfirmationUIModule from '../../../src/components/download_list/DownloadConfirmationUI'
 import { DownloadConfirmationUIProps } from '../../../src/components/download_list/DownloadConfirmationUI'
 import { FolderDownloadConfirmation } from '../../../src/components/download_list'
-import { SynapseClientError } from 'synapse-client/uti'
```

Comment:
This is just used as a type in a non-type-checked test file, which explains why this passed in CI